drivers/adc/ad738x/ad738x.c

```diff
@@ -351,6 +351,7 @@ int32_t ad738x_init(struct ad738x_dev **device,
 #endif
 	dev->conv_mode = init_param->conv_mode;
 	dev->ref_sel = init_param->ref_sel;
+	dev->ref_voltage = init_param->ref_voltage;
```

Comment:
I would move the change from ad738x driver, regarding the ref_voltage in a different commit. It is not really related to this commit about IIO support.